### ISSUE TITLE
full screen issue

### ISSUE THREAD

**Issue by hasanm08** [balvinderz/video_player_web_hls#25]
i cannot use full screen on web
```
Height of Platform View type: [videoPlayer-1] may not be set. Defaulting to `height: 100%`.
Set `style.height` to any appropriate value to stop this message.
Width of Platform View type: [videoPlayer-1] may not be set. Defaulting to `width: 100%`.
Set `style.width` to any appropriate value to stop this message.
```
some times reported

**Comment by balvinderz** [comment]
can you reproduce this with the latest version of the plugin?

**Comment by hasanm08** [comment]
> can you reproduce this with the latest version of the plugin?

my version is currently updated ```0.1.11+3```

**Comment by balvinderz** [comment]
Fixed in 0.1.11+4
